pkg/api/v1/types.go

```diff
@@ -55,10 +55,10 @@ const SubresourceGroupName = "subresources.kubevirt.io"
 const DefaultGracePeriodSeconds int64 = 30
 
 // GroupVersion is group version used to register these objects
-var GroupVersion = schema.GroupVersion{Group: GroupName, Version: "v1alpha2"}
+var GroupVersion = schema.GroupVersion{Group: GroupName, Version: "v1alpha3"}
```

Comment:
Just for curiosity, when do we decide to increase API version?

Reply:
I increase it when it is incommpatible with previous VMI definitions. `v1alpha2` definitions will not work anymore after this change.